aspnetcore/includes/2.1.md

```diff
@@ -0,0 +1 @@
+ASP.NET Core 2.1.0-preview1 is in preview and not recommended for production use.
```

Comment:
How about: ASP.NET Core 2.1 is in preview and not recommended for production use at this time.